app/services/outbound_health_checker.rb

```diff
@@ -0,0 +1,37 @@
+# Checks outbound network connections
+module OutboundHealthChecker
+  module_function
+
+  # @return [HealthCheckSummary]
+  def check
+    HealthCheckSummary.new(healthy: true, result: outbound_response)
+  rescue StandardError => err
+    NewRelic::Agent.notice_error(err)
+    HealthCheckSummary.new(healthy: false, result: err.message)
+  end
+
+  def outbound_response
+    if AppConfig.env.outbound_connection_check_url.blank?
+      raise 'missing outbound_connection_check_url'
+    end
+
+    response = faraday.get(AppConfig.env.outbound_connection_check_url)
```

Comment:
nitpick: If the response is not important, a `HEAD` request can be issue instead.
```suggestion
    response = faraday.head(AppConfig.env.outbound_connection_check_url)
```

Of course, that would need to change the `stub_request` on the specs.

Reply:
If we can, that makes sense to me. The HEAD request works for me locally, and we don't use the body, so I don't see why not?

Reply:
Sure, will switch to HEAD